openlibrary/macros/SearchResultsWork.html

```diff
@@ -3,6 +3,7 @@
 $code:
   max_rendered_authors = 9
   doc_type = (
+    'solr_work' if doc.get('type') == 'work' else
```

Comment:
It is indeed. How strange. I had thought this would be the case, and I would have sworn to you I had tested this repeatedly and found it not to be so.